core/src/main/scala/com/stripe/dagon/HCache.scala

```diff
@@ -6,16 +6,6 @@ package com.stripe.dagon
 sealed class HCache[K[_], V[_]] private (init: HMap[K, V]) {
   private var hmap: HMap[K, V] = init
 
-  /**
```

Comment:
any real reason to remove these? They seem valuable to me and very easy to add to the API?